### ISSUE TITLE
gulp build failure

### ISSUE THREAD

**Issue by andyfleming** [plouc/mozaik#56]
Running `gulp build` gives me the following. I'm using the install/setup instructions in the README.

```
module.js:340
    throw err;
    ^

Error: Cannot find module 'mozaik/gulpfile'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:289:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/path/to/gulpfile.js:5:10)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:313:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

**Comment by plouc** [comment]
@andyfleming, you used the yeoman generator ?

**Comment by andyfleming** [comment]
Yes (if I recall correctly).

I tried to just follow the directions in the README.

I was able to clone that other example repository to get things working but the `yo` workflow didn't work.

**Comment by plouc** [comment]
I've fixed a problem with the generator (v1.0.1)

**Comment by plouc** [comment]
@andyfleming, I close this issue as it seems to work now, feel free to re-open if it doesn't work for you. And thanks for the feedback!
